CRM/Utils/System/Base.php

```diff
@@ -718,7 +718,14 @@ public function getCiviSourceStorage() {
       // the system for a loop on lobo's macosx box
       // or in modules
       $cmsPath = $config->userSystem->cmsRootPath();
-      $userFrameworkResourceURL = $baseURL . str_replace("$cmsPath/", '',
+      $realCmsPath = realpath($cmsPath);
+
+      if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
+        $cmsPath = strtolower(str_replace('\\', '/', $cmsPath));
+        $realCmsPath = strtolower(str_replace('\\', '/', $realCmsPath));
+      }
+
+      $userFrameworkResourceURL = $baseURL . str_replace(["$cmsPath/", "$realCmsPath/"], '',
           str_replace('\\', '/', $civicrm_root)
```

Comment:
With the strtolower added, there'd be a mismatch on line 729 where it compares to $civicrm_root, which might contain a capital C. It should work to change it to `strtolower($civicrm_root)` on line 729.

Reply:
You're right, thanks.